dojoWordle/components/LetterBox.tsx/LetterBox.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Text } from "react-native";
+import { Box } from "./LetterBox.style";
+
+type letterStatus = "success" | "failure" | "missed";
+
+export type LetterBoxProps = {
+  status: letterStatus;
+};
+
+export function LetterBox({ status }: LetterBoxProps) {
+  return (
+    <Box status={status}>
+      <Text> A </Text>
```

Comment:
It's weird for me to handle padding/margin via spaces.

You should set spacing in Box style

Reply:
Yes but the spacer are for a row of those, I will make this style later on

Reply:
I mean removing " A " to have "A" and using padding in Box component.

But we can see that later if you have another idea